### DISCUSSION TITLE
[Diamond] Feature Request - RTL Layout

### DISCUSSION THREAD

**Discussion by mekhaba** [primefaces/community#47] [PrimeFaces Templates] [answered]
Hi Primefaces team, 

I would like to check if is there any update in the future to add RTL layout for DIamon theme. 

RTL layout is very important for our clients who use Arabic language. 

I'm very interested for this theme and if you add this feature I will be the first one who buy it. 

Looking forward to hearing soon. 

Best regards.

**Comment by atkntepe** [comment, marked as answer]
Hi, for Primeces Templates, our initial plan is to update them to PF13. After that, we will work on a new plan for our templates and what features we will add. Diamond is one of our next-gen templates, so if we add RTL, Diamond will definitely get that addition. But at the moment, I can't give you a definitive answer for Diamond RTL. When our planning is done for the templates, we will announce it.

Best regards.

**Comment by mekhaba** [comment]
Ok, understood and good luck for you . 
thank you.

**Comment by tandraschko** [comment]
we decided to not enhance existing templates with such big features, instead we will working on new templates soon.
